scss/_patterns_divider.scss

```
@@ -28,11 +28,11 @@
     @media (min-width: $threshold-6-12-col) {
       &:not(:nth-child(1))::before {
```

While we are at it, can it be simplified to:

```suggestion
      &:not(:first-child)::before {
```